Conditionals .ipynb

```diff
@@ -0,0 +1,152 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a2f94229-a424-45da-9988-1c4db903b5fb",
+   "metadata": {},
+   "source": [
+    "# Conditional Statements\n",
+    "\n",
+    "## if Statement\n",
+    "\n",
+    "It is used to decide whether a certain statement or block of statements will be executed or not.\n",
+    "\n",
+    "`if conditional_expr:`<br>\n",
+    "`    statement`\n",
+    "\n",
+    "    \n",
+    "In a Python program, contiguous statements that are indented to the same level are considered to be part of the same block.    \n",
+    "    \n",
+    "`if conditional_expr:`<br>\n",
+    "`    statement`<br>\n",
+    "`    statement`<br>\n",
+    "`    ...`<br>\n",
+    "`    statement`<br>\n",
+    "`following_statement`\n",
+    "    \n",
+    "## The else and elif Clauses\n",
+    "\n",
+    "The  **else** clause is used to specify an alternative path when the expression is not True\n",
+    "\n",
+    "`if conditional_expr:`<br>\n",
+    "`    statement(s)`<br>\n",
+    "`else:`<br>\n",
+    "`    statement(s)`\n",
+    "        \n",
+    "The  **elif** clauses is used  for branching execution based on several alternatives   \n",
+    "    \n",
+    "`if conditional_expr_1:`<br>\n",
+    "`    statemen(s)`<br>\n",
+    "`elif conditional_expr_2:`<br>\n",
+    "`    statement(s)`<br>\n",
+    "`elif conditional_expr_3:`<br>\n",
+    "`    statement(s)`<br>\n",
+    "`else:`<br>\n",
+    "`    statement(s)`\n",
+    "    \n",
+    "## Python’s Ternary Operator\n",
+    "\n",
+    "It allows testing a condition in a single line replacing the multiline if-else making the code compact. \n",
+    "\n",
+    "`var = value_1 if conditional_expr else value_2`\n",
+    "\n",
+    "## Examples"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "72e33b88-5414-4ac3-a8b3-5f3d92270bcd",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "10 is greater than 5\n"
+     ]
+    }
+   ],
+   "source": [
+    "a = 10\n",
+    "b = 5\n",
+    "\n",
+    "if a > b:\n",
+    "    print(a, \"is greater than\", b)\n",
+    "elif(a < b):\n",
```

Comment:
remove the parenthesis, but try to mention that they could be used whenever they're required.